src/layout/navigation/Navigation.tsx

```diff
@@ -149,7 +149,12 @@ const Navigation: React.FC<NavigationProps> = ({
                   }}
                 />
                 <NavItem backgroundColor={backgroundColor}>
-                  <SupportButton to="/support-us">Support Pride</SupportButton>
+                  <SupportButton
+                    onClick={() => setOpen(!isOpen)}
+                    to="/support-us"
+                  >
+                    Support Pride
+                  </SupportButton>
```

Comment:
Slightly odd behaviour when this button is selected from the mobile menu when on the `/support-us` page. I think we can safely use `false` here as this button will never be used to open the nav.